src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -339,26 +336,11 @@ private void WaitForRunCompletion()
 
         private void SelectTabByGuid(Guid guid)
         {
-            // If guid is Empty - switch to HomeWorkspace
-            if (guid.Equals(Guid.Empty) && dynamoModel.CurrentWorkspace != dynamoModel.HomeSpace)
-            {
-                dynamoModel.Home(null);
```

Comment:
as I discovered while merging, switching to Home ws functionality was moved onto View layer that can't be used by viewless DynamoWebServer